app/extensions/edm/__init__.py

```diff
@@ -291,6 +293,52 @@ def _get(
         return response
 
 
+class EDMObjectMixin(object):
+    def _process_edm_attribute(self, data, edm_attribute):
+        edm_attribute = edm_attribute.strip()
+        edm_attribute = edm_attribute.strip('.')
+        edm_attribute_list = edm_attribute.split('.')
```

Comment:
does python support chaining these things? ie
```
edm_attribute_list = edm_attribute.strip().strip('.').split('.')
```
if so might be a little more readable that way

Reply:
I agree.  I do this type of code when I may want to easily turn off a given pre-processing line.  It makes it very easy to comment it out, but it adds lines and makes it a bit harder to read.